OpenEdXMobile/res/values/labels.xml

```diff
@@ -42,4 +42,6 @@
     <string name="label_undo">Undo</string>
     <!-- Label denoting an error -->
     <string name="label_error">Error</string>
+    <!-- Customer support text, using as button title for disabled user-->
+    <string name="CUSTOMER_SUPPORT">Customer Support</string>
```

Comment:
please change the name string into small letters

Reply:
update to `label_customer_support`